### PR TITLE
Update zh-CN.json Chinese translation

### PR DESCRIPTION
**Pull request by uos-ljtian**

### CODE REVIEW

File: translations/zh-CN.json [hunk 1]
```diff
@@ -522,28 +522,28 @@
...
 	"Multiple minikube profiles were found -": "发现了多个 minikube 配置文件 -",
 	"Multiple minikube profiles were found - ": "找到多个 minikube 配置文件 - ",
 	"NIC Type used for host only network. One of Am79C970A, Am79C973, 82540EM, 82543GC, 82545EM, or virtio (virtualbox driver only)": "网卡类型仅用于主机网络。Am79C970A, Am79C973, 82540EM, 82543GC, 82545EM 之一，或 virtio(仅限 VirtualBox 驱动程序)",
-	"NIC Type used for nat network. One of Am79C970A, Am79C973, 82540EM, 82543GC, 82545EM, or virtio (virtualbox driver only)": "",
-	"NOTE: Please do not close this terminal as this process must stay alive for the tunnel to be accessible ...": "",
-	"NOTE: This process must stay alive for the mount to be accessible ...": "",
+	"NIC Type used for nat network. One of Am79C970A, Am79C973, 82540EM, 82543GC, 82545EM, or virtio (virtualbox driver only)": "用于 nat 网络的 NIC 类型。 Am79C970A、Am79C973、82540EM、82543GC、82545EM 或 virtio 之一（仅限 virtualbox 驱动程序）",
+	"NOTE: Please do not close this terminal as this process must stay alive for the tunnel to be accessible ...": "注意：请不要关闭此终端，因为此进程必须保持活动状态才能访问隧道......",
+	"NOTE: This process must stay alive for the mount to be accessible ...": "注意：此进程必须保持活动状态才能访问安装......",
```
Comment (liangyuanpeng) [inline translations/zh-CN.json:537]: 应该保持一致的三个省略号?
见上面的    将主机路径 {{.sourcePath}} 挂载到虚拟机中作为 {{.destinationPath}} ...

File: translations/zh-CN.json [hunk 2]
```diff
@@ -522,28 +522,28 @@
...
-	"Multiple errors encountered:": "",
+	"Multiple errors encountered:": "遇到了多个错误：",
 	"Multiple minikube profiles were found -": "发现了多个 minikube 配置文件 -",
 	"Multiple minikube profiles were found - ": "找到多个 minikube 配置文件 - ",
 	"NIC Type used for host only network. One of Am79C970A, Am79C973, 82540EM, 82543GC, 82545EM, or virtio (virtualbox driver only)": "网卡类型仅用于主机网络。Am79C970A, Am79C973, 82540EM, 82543GC, 82545EM 之一，或 virtio(仅限 VirtualBox 驱动程序)",
-	"NIC Type used for nat network. One of Am79C970A, Am79C973, 82540EM, 82543GC, 82545EM, or virtio (virtualbox driver only)": "",
-	"NOTE: Please do not close this terminal as this process must stay alive for the tunnel to be accessible ...": "",
-	"NOTE: This process must stay alive for the mount to be accessible ...": "",
+	"NIC Type used for nat network. One of Am79C970A, Am79C973, 82540EM, 82543GC, 82545EM, or virtio (virtualbox driver only)": "用于 nat 网络的 NIC 类型。 Am79C970A、Am79C973、82540EM、82543GC、82545EM 或 virtio 之一（仅限 virtualbox 驱动程序）",
+	"NOTE: Please do not close this terminal as this process must stay alive for the tunnel to be accessible ...": "注意：请不要关闭此终端，因为此进程必须保持活动状态才能访问隧道......",
```
Comment (liangyuanpeng) [inline translations/zh-CN.json:536]: 应该保持一致的三个省略号?

File: translations/zh-CN.json (hunk 1 above)
Comment (spowelljr) [reply translations/zh-CN.json:537]: Hi @liangyuanpeng, thank you for your translation review. I'm not sure if @uos-ljtian is going to check back on this PR. So feel free to use the `Add a suggestion` button (shown below) and enter what you'd change and I can accept your suggestion into the PR and merge it.

Screenshot 2024-02-01 at 3 33 43 PM

File: translations/zh-CN.json (hunk 2 above)
Comment (spowelljr) [reply translations/zh-CN.json:536]: Same as above

**Pull request merged**